sites/partners/cypress/integration/admin-user-management.spec.ts

```diff
@@ -7,6 +7,35 @@ describe("Admin User Mangement Tests", () => {
     cy.signOut()
   })
 
+  it("as admin user, should show all users", () => {
```

Comment:
Nice catch on the typo of the file name. I wonder if these tests should live in this file and the listing export test below should live in a different file since it's not really "user" management. I think if I was adding the listing export tests in bloom core I would put it in the `03-listing.spec.ts` file. Thoughts?

Reply:
Ohhh I see your point! I was originally interpreting it more as "admin user"... management if that makes sense lol but I'm good with that change and have updated the PR!